.ci/37.yml

```diff
@@ -16,4 +16,10 @@ dependencies:
   - scipy
   - libpysal
   - tqdm
+  - coveralls
```

Comment:
Since `codecov` is used in `uni tests.yml`, shouldn't it be used here and not `coveralls`?